INSTRUMENTS/Serial/Serial/SERIAL.py

```diff
@@ -0,0 +1,95 @@
+from flojoy import flojoy, DataContainer
+from time import time, sleep
+import serial
+import numpy as np
+from datetime import datetime
+import plotly.express as px
+
+@flojoy
+def SERIAL(v, params):
+    '''
+    Node to take simple time dependent 1d data from an Ardunio, or a similar serial device.
+    For example you can record temperature following this tutorial:
+
+    https://learn.adafruit.com/thermistor/using-a-thermistor
+
+    with Serial.println(steinhart) as the only line printing.
+
+    It is important that the last line Arduino is returning is the data with a new line at the end (i.e. println()).
+
+    The other lines must be returned with print() with print(",") between each line. 
+
+    For example:
+
+    print(reading0)
+    print(",")
+    println(reading1)
+    '''
+    print('parameters passed to SERIAL: ', params)
+    COM_PORT = params.get('com_port', '/dev/ttyUSB0') # The COM port to use (e.g.: Linux/Mac: '/dev/ttyUSB0' or Windows: COM1).
+    BAUD = int(params.get('BAUD_RATE', 9600))
+    NUM = int(params.get('num_readings', 100)) # Number of points to record. Length of recording ~= NUM * RECORD_PERIOD seconds.
+    RECORD_PERIOD = float(params.get('record_period', 1)) # Period of recordings in seconds (i.e. the time between two recordings).
+    # NUM = 5  # Debug
```

Comment:
delete uneccesary comments

Reply:
Debugging comments and print statements removed.